This doesn't build against linux-yocto:
```
2026-04-16 09:37:30 - INFO     - | video/../video/driver/vidc/src/firmware.c: In function 'fw_pas_load':
2026-04-16 09:37:30 - INFO     - | video/../video/driver/vidc/src/firmware.c:47:15: error: implicit declaration of function 'devm_qcom_scm_pas_context_alloc' [-Wimplicit-function-declaration]
2026-04-16 09:37:30 - INFO     - |    47 |         ctx = devm_qcom_scm_pas_context_alloc(dev, pas_id, phys, res_size);
2026-04-16 09:37:30 - INFO     - |       |               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2026-04-16 09:37:30 - INFO     - | video/../video/driver/vidc/src/firmware.c:47:13: error: assignment to 'struct qcom_scm_pas_context *' from 'int' makes pointer from integer without a cast [-Wint-conversion]
2026-04-16 09:37:30 - INFO     - |    47 |         ctx = devm_qcom_scm_pas_context_alloc(dev, pas_id, phys, res_size);
2026-04-16 09:37:30 - INFO     - |       |             ^
2026-04-16 09:37:30 - INFO     - | video/../video/driver/vidc/src/firmware.c:53:12: error: invalid use of undefined type 'struct qcom_scm_pas_context'
2026-04-16 09:37:30 - INFO     - |    53 |         ctx->use_tzmem = core->fw.dev;
```

--- a/recipes-kernel/iris-video-module/iris-video-dlkm_1.0.5.bb
+++ b/recipes-kernel/iris-video-module/iris-video-dlkm_1.0.5.bb
@@ -8,7 +8,7 @@ SRC_URI = " \
     file://blacklist-video.conf.venus \
     file://blacklist-video.conf.vidc \
 "
-SRCREV  = "02248f4021f7690e675c408b4fd0208ad559a1d5"
+SRCREV  = "4e5e930893ba95e6684b69fcad8db33999fd2412"
 
 inherit module update-alternatives
 

